examples/annotated-http-service/src/main/java/example/armeria/server/annotated/ExceptionHandlerService.java

```diff
@@ -0,0 +1,107 @@
+package example.armeria.server.annotated;
+
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.common.logging.LogLevel;
+import com.linecorp.armeria.server.HttpStatusException;
+import com.linecorp.armeria.server.annotation.ExceptionHandler;
+import com.linecorp.armeria.server.annotation.ExceptionHandlerFunction;
+import com.linecorp.armeria.server.annotation.Get;
+import com.linecorp.armeria.server.annotation.Param;
+import com.linecorp.armeria.server.annotation.decorator.LoggingDecorator;
+
+import example.armeria.server.annotated.ExceptionHandlerService.GlobalExceptionHandler;
+
+/**
+ * Examples how to use {@link ExceptionHandler}.
+ */
+@LoggingDecorator(
+        requestLogLevel = LogLevel.INFO,            // Log every request sent to this service at INFO level.
+        successfulResponseLogLevel = LogLevel.INFO  // Log every response sent from this service at INFO level.
+)
+@ExceptionHandler(GlobalExceptionHandler.class)
+public class ExceptionHandlerService {
+
+    /**
+     * Throws a {@link LocallySpecificException} which can be handled by the {@link LocalExceptionHandler}.
+     */
+    @Get("/locallySpecific")
+    @ExceptionHandler(LocalExceptionHandler.class)
+    public String exception1() {
+        throw new LocallySpecificException();
+    }
+
+    /**
+     * Throws a {@link LocallyGeneralException} which can be handled by the {@link LocalExceptionHandler}.
+     */
+    @Get("/locallyGeneral")
+    @ExceptionHandler(LocalExceptionHandler.class)
+    public String exception2() {
+        throw new LocallyGeneralException();
+    }
+
+    /**
+     * Throws a {@link GloballyGeneralException} which can be handled by the {@link GlobalExceptionHandler}.
+     */
+    @Get("/globallyGeneral")
+    @ExceptionHandler(LocalExceptionHandler.class)
+    public String exception3() {
+        throw new GloballyGeneralException();
+    }
+
+    /**
+     * Throws an {@link IllegalArgumentException} which can be handled by the default exception handler.
```

Comment:
I'm wondering whether we need to tell the reader more about default exception handlers such as what kinds of exceptions they handles, and what happens if there's a specific exception that default exception handlers cannot handle.
Probably overkill?

Reply:
They are explained in our official web site(https://line.github.io/armeria/server-annotated-service.html). So I think it would be better to add links to the documents. Thank you for the suggestion!